src/test/scala/kamon/opentracing/TracerSpec.scala

```diff
@@ -0,0 +1,121 @@
+package kamon.opentracing
+
+import java.nio.ByteBuffer
+
+import io.opentracing.propagation.{Format => OpenFormat}
+import kamon.opentracing.test.{Harness, TestTextMap}
+import kamon.trace.ActiveSpan.{Default => KamonDefaultActiveSpan}
+import kamon.trace.IdentityProvider.{Identifier => KamonIdentifier}
+import kamon.trace.SpanContext.{Baggage => KamonBaggage, SamplingDecision => KamonSamplingDecision, Source => KamonSource}
+import kamon.trace.{SpanContext => KamonSpanContext, SpanContextCodec => KamonSpanContextCodec}
+import kamon.util.HexCodec
+import org.scalatest.DoNotDiscover
+
+@DoNotDiscover
+class TracerSpec extends Harness {
+  var tracer: Tracer = _
+
+  private def initId(id: Long): KamonIdentifier = {
+    val data = ByteBuffer.wrap(new Array[Byte](8))
+    data.putLong(id)
+    KamonIdentifier(HexCodec.toLowerHex(id), data.array())
+  }
+
+  "Tracer" should {
+    "be available via a no-arg constructor" in {
+      tracer = new Tracer
+      tracer must be (a[Tracer])
```

Comment:
if it walks like a duck, it looks like a duck and you just created a instance with the very specific type you are asserting in the line above it, it must be a duck :D.. this test seems unnecessary.